src/create-route-map.js

```diff
@@ -34,6 +34,16 @@ export function createRouteMap (
     }
   }
 
+  // warn if routes do not include leading slashes
+  const pathsMissingSlashes = pathList
+    .filter(path => !!path && path.charAt(0) !== '*' && path.charAt(0) !== '/')
```

Comment:
I think a single `find` is simpler. No need for the map and I don't really understand the last `filter` call
We can also remove the `!!path` check as it's a string

Reply:
`find` is much simpler but unfortunately I do not think it's supported by this version of nightwatch which causes e2e tests to fail.

Reply:
you are totally right! Thank you for catching that, it would fail on IE as well. Then let's keep the `filter` and display the list of failing routes since the information is there already. If the routes could be displayed one per line, that would be perfect as it would be more readable (I don't remember how it was before)

Reply:
It used to be just comma separated, but one per line sounds better